server/controllers/recommendation/matchIndexOffenceController.test.ts

```diff
@@ -144,6 +146,7 @@ describe('post', () => {
         bookRecallToPpud: {
           policeForce: 'Kent',
           indexOffence: 'some offence',
+          indexOffenceComment: null,
```

Comment:
Can you add tests for the empty string and whitespace cases too, please, to make sure it's setting `indexOffenceComment` to `null` when expected?